receiver/mongodbreceiver/README.md

```diff
@@ -61,6 +60,13 @@ The full list of settings exposed for this receiver are documented [here](./conf
 
 ## Metrics
 
+The following metric are available with versions:
+- `mongodb.extent.count` < 4.4 with mmapv1 storage engine
+- `mongodb.session.count` >= 3.0 with wiredTiger storage engine
+- `mongodb.cache.operations` >= 3.0 with wiredTiger storage engine
+- `mongodb.connection.count` with attribute `active` >= 4.0
+- `mongodb.index.access.count` >= 4.0
+
```

Comment:
This is a nice improvement, but it's unrelated to fixing the partial errors. Please submit as a separate PR.

Reply:
will add in follow up pr